docker-compose.yml

```diff
@@ -0,0 +1,59 @@
+version: "3.9"
+services:
+  question-service:
+    build: ./backend/question-service
+    ports:
+      - 8080:8080
+    volumes:
+      - ./backend/question-service:/usr/src/app
+
+  frontend:
+    build: ./frontend
+    volumes:
+      - ./frontend:/app
+    ports:
+      - 3000:3000
+    env_file: ./frontend/.env
+
+  user-service:
+    build: ./backend/user-service
+    ports:
+      - 8000:8000
+    volumes:
+      - ./backend/user-service:/usr/src/app
+    depends_on:
+      - postgres
+    env_file: ./backend/user-service/.env.docker
+
+  postgres:
+    image: postgres
+    restart: always
+    environment:
+      POSTGRES_DB: peerprep
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready -U postgres" ]
+      interval: 1m30s
+    ports:
+      - 5432:5432
+
+  migration:
+    build: ./backend/user-service
+    depends_on:
+      postgres:
+        condition: service_healthy
+    command:
+      [
+        "npx",
+        "prisma",
+        "db",
+        "push",
+        ";",
+        "npm",
+        "run",
+        "seed"
+      ]
```

Comment:
hmm im not entirely sure about this, but is the intended behavior - each time we run `docker-compose run migration;`, we seed the data? 
```
command:
     [
        "npx",
        "prisma",
        "db",
        "seed",
        "npx",
        "&&",
        "prisma",
        "db",
        "push",
        ";"
      ]
```